<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgScripts</class>
 <widget class="QDialog" name="CQDlgScripts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripts</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QToolButton" name="qqAddNewScript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Insert new script</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Script properties</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="qqExplicitHandling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Explicit handling</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqThreaded">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Script launches a new thread</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqExecuteOnce">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Execute just once</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqExecuteInSensingPhase">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Execute in the sensing phase</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Associated object</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqAssociatedObjectCombo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Execution order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqExecutionOrder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="qqScriptList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
